### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by cbarbozaerasmus**
Hello,

- Checks are implemented for TrainExplore.
- I am doing the main checks first, since you call settingsExplore from there. 
- I commented    ClassFeature, PositiveClass, FeatureInclude, Maximize, Accuracy, Specificity, since I have problems running the tests with the default values we discussed. But it is just matter to define the correct ones.

### CODE REVIEW

**Comment by AniekMarkus** [comment]
ClassFeature, PositiveClass -> should always be given by the user, do not use a default value (we need to check the strings with/without '', so that the code can still run even if these are included in the column names)
FeatureInclude -> default is now "" -> this should be a valid option, does an empty string not pass the test? is there another way in which the user can decide not to use this functionality?
Maximize -> this seems to be working?
Accuracy, Specificity -> default values can be set to 0 values

**Comment by cbarbozaerasmus** [comment]
Hey!

FeatureInclude is working with an empty string. For now I can only imagine we can set that to NULL, and later convert that to “”, but if the empty string is the default, there’s not much difference.

I made those corrections and the tests passed. I am working now on making sure it passes most Github Actions and eliminate the warnings. Let’s see if I can finish that soon, at least by the end of this week.

I hope you are having a productive summer school!

Cheers.

**Comment by cbarbozaerasmus** [comment]
On this PR, R CMD Check is passing without errors. We still have a Warning pending.

**Comment by cbarbozaerasmus** [comment]
@AniekMarkus I also added a new rule in which we need an approval before merging to main.

File: R/MainFunctions.R
Comment (AniekMarkus) [inline R/MainFunctions.R:223]: We should change defaults to TRUE/FALSE here

**Review by AniekMarkus — APPROVED** [review]
No comments - ready to merge!

**Pull request merged**
